src/templates/filter-bar-controller.js

```diff
@@ -29,12 +32,20 @@ FilterBarController.prototype.selectFilter = function(filterName) {
 };
 
 FilterBarController.prototype.getFilteredNotes = function(notes) {
-    const activeTab = Filters[getFilterName(store.getState())];
+    const state = store.getState();
+    const filterName = getFilterName(state);
+    const activeTab = Filters[filterName];
     if (!notes || !activeTab) {
         return [];
     }
 
-    return notes.filter(activeTab().filter);
+    const filteredNoteReads = getFilteredNoteReads(state);
```

Comment:
do we ever need this list? what if it weren't stored as a list (as it probably won't always)

what if instead we had the selector return if it's a filtered read note?

```js
'unread' === filterName && isFilteredNoteRead( note.id )
``